### DISCUSSION TITLE
Regex Expression IN pin

### DISCUSSION THREAD

**Discussion by PrismMind** [FakeFishGames/Barotrauma#9070] [Ideas and suggestions]
What it says in the title, Regex needs a expression in pin so that the expression check can be assigned by other components.

**Comment by mLuby** [comment]
That would be nice. In the meantime it's possible to get close to it by concatenating the target string with the string to search and using a pattern that looks for the first capture group again. See this [Barotrauma Regex Cheatsheet](https://docs.google.com/document/d/1miqYsNJTFoh1W-MsRAH7m0gBt9up5ssXu1C1YFQxyRc/edit#heading=h.5mg26ff228sd). For extra help, talk to the folks in the official Discord's #sub-building channel.
